src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -160,6 +166,7 @@ public EditPersonDescriptor(EditPersonDescriptor toCopy) {
             setAddress(toCopy.address);
             setRoles(toCopy.roles);
             setCcas(toCopy.ccas);
+            setMetadata(toCopy.metadata);
```

Comment:
Github doesnt allow me to comment below but `isAnyFieldEdited` needs to be editted.